fiftyone/core/odm/utils.py

```
@@ -122,6 +136,61 @@ def deserialize_value(value):
+        return d
+
+    result = {}
+    for k, v in d.items():
+        if isinstance(v, dict):
+            # Check if this is an embedded document with a _cls field
+            if "_cls" in v:
+                try:
+                    # Try to deserialize it using our custom deserializer
+                    # which handles unknown types gracefully
+                    result[k] = deserialize_value(v)
+                except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)